README.md

```diff
@@ -0,0 +1,189 @@
+# FFMQ Randomizer
+
+This Final Fantasy Mystic Quest randomizer is only compatible with the NA v1.1 rom file (MD5: f7faeae5a847c098d677070920769ca2).
+
+## How to run
+
+### Linux
+
+Install the latest dotnet for your distro: https://docs.microsoft.com/en-us/dotnet/core/install/linux
+
+Run `dotnet run` in the FFMQWebAsm directory
+
+Open http://localhost:5000/ in your browser of preference
+
+## ROM knowledge
+
+### Raw Byte Info
+
+#### Enemies
```

Comment:
This info should probably be put in it's own document rather than filling the readme (something like docs/enemies.md), then the readme can point to it